plugins/basic/python/tk_maya_basic/plugin_logic.py

```diff
@@ -334,3 +345,40 @@ def _jump_to_signup():
     QtGui.QDesktopServices.openUrl(
         QtCore.QUrl("https://www.shotgunsoftware.com/signup")
     )
+
+
+class Callback(object):
```

Comment:
This maybe being over cautious but since we were using [Callback](https://github.com/LumaPictures/pymel/blob/c624e7e28697d611a6c6d027129c243b43b6c40e/pymel/internal/factories.py#L718) from Pymel, I wanted to replicate the similar behavior and also combine it with what we were doing in the menu generation.

Reply:
Good call. When refactoring code like this, I find that it often helps to leave the code as untouched as possible and try to replicate the original usage pattern of the class. This way, it's easier for the submitter and reviewer to figure out how the code works and if anything was missed during the conversion.